packages/react/src/page-alert/PageAlert.tsx

```diff
@@ -38,21 +43,26 @@ export type PageAlertProps = PropsWithChildren<{
 
 export const PageAlert = forwardRef<HTMLDivElement, PageAlertProps>(
 	function PageAlert(
-		{ id, role, children, onClose, onDismiss, title, tone, tabIndex },
+		{ id, isHidden, role, children, onClose, onDismiss, title, tone, tabIndex },
 		ref
 	) {
 		const closeHandler = getOptionalCloseHandler(onClose, onDismiss);
 		const { fg, bg, icon } = pageAlertToneMap[tone];
+
 		return (
 			<Flex
+				css={{
+					backgroundColor: bg,
+					position: 'relative',
+					...(isHidden && { display: 'none' }),
```

Comment:
This feels weird to me. I think I would prefer to see a `focusOnMount` prop, which would suit the majority of scenarios, including all of the error state messages that we have in the example app.

This scenario is probably still valid, and may be the only way to make it work with the Drawer/Modal returning focus, but can be achieved by wrapping it in a hidden `Box`

Reply:
So the Box receives focus rather than the alert? Something like:

```ts
<Box css={isHidden ? { display: 'none' } : undefined} focusAll ref={sectionAlertContainerRef} tabIndex={-1}>
  <SectionAlert ... />
</Box>

<Drawer elementToFocusOnClose={sectionAlertContainerRef.current} .. />
```

The thing I like about a CSS solution is that it will always work. Rather than `focusOnMount`* with caveat for drawer. I've never used a `focusOnMount` prop but how do you avoid a race condition when you have page and section alerts?

Reply:
One of our goals was to reduce the need for consumers to have to do this all the time throughout their apps. I feel a 95% automatic solution with a caveat for components which manage focus is ok. 
<img width="578" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/e0e5a6f1-6b78-423d-a1db-83516b68670d">

Your example could also be this:

```
<Box css={{display: isHidden ? 'none' : undefined }}>
  <SectionAlert ref={sectionAlertContainerRef} tabIndex={-1} />
</Box>

<Drawer elementToFocusOnClose={sectionAlertContainerRef.current} .. />
```

Reply:
>One of our goals was to reduce the need for consumers to have to do this all the time throughout their apps. I feel a 95% automatic solution with a caveat for components which manage focus is ok.

Interesting, yeah that makes sense, I like that.

So, would `focusOnMount` be an boolean? And if so, how would you manage page vs section alert race conditions?

Reply:
If you're rendering a page alert and a section alert, then you only want 1 to received focus, so it's up to you to manage that state of which would receive it. This is the same for a `focusOnMount` or an "is hidden" approach